### DISCUSSION TITLE
Lesson 7 testPriceFeedVersionIsAccurate() -> [FAIL. Reason: EvmError: Revert]

### DISCUSSION THREAD

**Discussion by MIKEX818** [Cyfrin/foundry-full-course-cu#1849] [General]
Hello everyone, I am trying to compile function testPriceFeedVersionIsAccurate() with established SEPOLIA URL from Alchemy, but I am constantly running error on the test. I will provide the code and terminal commands as context below. Thanks for your time! 
-> FundMeTest.t.sol
<img width="1728" alt="Screenshot 2024-06-17 at 10 50 48 AM" src="https://github.com/Cyfrin/foundry-full-course-f23/assets/169244435/4308d448-8847-4bcf-b1d9-670f969f92d6">
-> DeployFundMe.s.sol
<img width="1728" alt="Screenshot 2024-06-17 at 10 52 02 AM" src="https://github.com/Cyfrin/foundry-full-course-f23/assets/169244435/65d767b6-559c-4673-9630-7d16097c3c7c">
-> Here is my terminal and error message
<img width="1408" alt="Screenshot 2024-06-17 at 10 53 25 AM" src="https://github.com/Cyfrin/foundry-full-course-f23/assets/169244435/ad8117de-b77a-4d27-9819-777e3af4d3cf">

**Comment by urosognjenovic** [comment]
The function ```getVersion``` calls the ```version``` function at the price feed address. In your case, you didn't create the mock contract so you will make a call to a nonexistent contract address.

You didn't create the ```HelperConfig.s.sol``` which will create a ```MockV3Aggregator``` of the price feed when forking the Sepolia testnet. When you do create ```HelperConfig.s.sol```, the function ```getVersion``` will call the ```version``` function of the mock contract. 

I recommend running the command on your local Anvil chain, ie. without the ```--rpc-url```.
